recipes/DateTimeRange/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "DateTimeRange" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DateTimeRange-{{ version }}.tar.gz
+  sha256: 47bc10e5df23bb1f0d3ee493a26cb8d1d99e179884aabff62d3e51033b6a73ce
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - releasecmd
```

Comment:
```suggestion
```
Do you really need releasecmd? it seems like it has a graceful failure if it isn't installed.

Reply:
I'll wait to see what @thombashi thinks.